File: src/main/java/com/procurement/storage/StorageApplication.java
package com.procurement.storage;

import com.procurement.storage.config.ApplicationConfig;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.autoconfigure.liquibase.LiquibaseAutoConfiguration;

@SpringBootApplication(scanBasePackageClasses = {ApplicationConfig.class},
    exclude = {LiquibaseAutoConfiguration.class})
public class StorageApplication {
    public static void main(final String[] args) {
        SpringApplication.run(StorageApplication.class, args);
    }
}

File: src/main/kotlin/com/procurement/storage/StorageApplication.kt
package com.procurement.storage

import com.procurement.storage.config.ApplicationConfig
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cloud.netflix.eureka.EnableEurekaClient

@SpringBootApplication(scanBasePackageClasses = [ApplicationConfig::class])
@EnableEurekaClient
class StorageApplication

fun main(args: Array<String>) {
    runApplication<StorageApplication>(*args)
}
